app/src/components/v-table/table-row.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<tr class="table-row" :class="{ subdued: subdued, clickable: hasClickListener }" @click="$emit('click', $event)">
+	<tr class="table-row" :class="{ subdued: subdued, clickable: hasClickListener }">
```

Comment:
Removing this click handler prevents users from clicking through to the item detail page from the collection overview. (for all collections)
![firefox_sT3jvyZGyf](https://user-images.githubusercontent.com/9389634/216306336-a071323f-b1ab-4e4e-aa56-3d2ee681e1cd.gif)

Reply:
Ah right, sorry about that, I forgot to add it back, when I merged the row into v-table since it already had a @click listener on the v-table level for each row.

Reply:
That being said, one option would be to to add a single click event on the table row, and use the $event.target property to pickup which row was clicked. This should in essence give us back the .2s. Shall I try it? @br41nslug

Reply:
So after running tests again, and then again, because it didn't make sense, I have concluded that at least when doing it with vue, it isn't faster or more performant implementing event delegation. It doesn't make sense, but either vue is doing some optimisations on v-for that I am unaware of but these are weird results:

```

Per row listener on tr
2.91
3.12
3.07

Single listener on tbody
4.07
5.33
4.09
```

Reply:
What exactly did you test? The broader the click handler gets the slower it becomes, not considering browser optimizations, presumably an event handler on `body` would get propagated through all its children to make sure it catches the child events that need to be bubbled up. So theoretically the more children there are the slower the single handler gets.

Reply:
Well this was initially what triggered my attempt for this: https://jsperf.app/jquery-event-delegation/120
But I guess event delegation vs just adding one listener to tbody is different somehow.